### PR TITLE
swirc: update to 3.4.0.

### PR DESCRIPTION
**Pull request by uhlin**
Swirc 3.4.0 out

#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, **x86_64-musl**
- I built this PR locally for these architectures:
  - armv6l-musl **cross**

### CODE REVIEW

File: srcpkgs/swirc/template [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
- $(vopt_if notify libnotify-devel)"
+makedepends="gettext-devel hunspell-devel libcurl-devel libidn-devel
+ ncurses-devel openssl-devel $(vopt_if notify libnotify-devel)"
 checkdepends="cmocka-devel"
 short_desc="Curses ICB and IRC client"
 maintainer="Markus Uhlin <markus.uhlin@bredband.net>"
 license="BSD-3-Clause, ISC, MIT"
 homepage="https://www.nifty-networks.net/swirc"
 changelog="https://raw.githubusercontent.com/uhlin/swirc/master/CHANGELOG.md"
 distfiles="https://www.nifty-networks.net/swirc/releases/${pkgname}-${version}.tgz"
```
Comment (classabbyamp) [inline srcpkgs/swirc/template:19]:
```suggestion
distfiles="https://www.nifty-networks.net/swirc/releases/swirc-${version}.tgz"
```

Comment (uhlin) [reply srcpkgs/swirc/template:19]: Can I commit the change here at GitHub or shall I do a force push? (There's a button "commit suggestion".)

Comment (classabbyamp) [reply srcpkgs/swirc/template:19]: force push is probably less steps

Comment (uhlin) [reply srcpkgs/swirc/template]: Ok, done

**Pull request merged**